source/blog/2016-08-16-vagrant-service-manager-1-3-0-release.html.md

```diff
@@ -0,0 +1,67 @@
+---
+title: vagrant-service-manager 1.3.0 Released - Focus on Kubernetes Support
```

Comment:
This title is a little too long and should state the official name and not the package name, something like:

"Vagrant Service Manager 1.3.0 Released"